styles/style.css

```diff
@@ -0,0 +1,237 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 30px;
+    font-family: 'Montserrat', sans-serif;
+    color: black;  
+}
+
+
+.page {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    background-image: url(../img/QuizBackground.png);
+    background-size: cover;
+    padding: 3%;
+}
+
+.menu{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 2%;
+}
+
+.menu__logo {
+    width: 25%;
+}
+
+.menu__list {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 65%;
+    height: 110px;
+    background-color: white;
+    border-radius: 16px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+}
+
+.menu__item {
+    position: relative;
+    padding: 10px;
+    width: 20%;
+    text-decoration: none;
+    font-weight: 600;
+    text-align: center;
+    color: black;
+    
+}
+
+.menu__item--selected::before {
+    content: "";
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 6px;
+    background: rgba(255, 0, 0, 0.5);
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 4px 4px 40px rgba(255, 0, 0, 0.9);
+    border-radius: 4px;
+}
+
+.menu__item--unselected {
+    color: rgba(0, 0, 0, 0.4);
```

Comment:
jako że usuwamy --unselected całkowicie to tu zmieńmy na --selected i dajmy `color: black;` a w podstawowym wariancie (linia 52) daj to wyszarzenie `color: rgba(0, 0, 0, 0.4);` zamiast `color: black;`

Reply:
ale unselected wg projektu jest własnie szare, a selected czarne plus ten before. Nie rozumiem, po co zamiana?

Reply:
bo zgodnie z komentarzem wyżej usuwamy klasę unselected :) zostaje stan domyślny - wyszarzony napis i stan --selected który ma mieć color: black

Reply:
nie zrozumiałam o co Ci chodziło, ale koniec końców wczoraj i tak zrobiłam tak jak napisałeś ;)